Source/RedTechArtToolsEditor/RedTechArtToolsEditor.build.cs

```diff
@@ -26,7 +26,7 @@ public class RedTechArtToolsEditor : ModuleRules
 {
 	public RedTechArtToolsEditor(ReadOnlyTargetRules Target) : base(Target)
 	{
-		PublicDependencyModuleNames.AddRange(new string[] { });
+		PublicDependencyModuleNames.AddRange(new string[] { "Engine", "PropertyEditor" });
```

Comment:
Rider through this in when fixing up the headers.  Didn't seem to hurt and seemed technically correct